bitwarden_license/src/Commercial.Core/AdminConsole/Providers/RemoveOrganizationFromProviderCommand.cs

```diff
@@ -75,13 +84,44 @@ public class RemoveOrganizationFromProviderCommand : IRemoveOrganizationFromProv
 
         await _stripeAdapter.CustomerUpdateAsync(organization.GatewayCustomerId, customerUpdateOptions);
 
-        var subscriptionUpdateOptions = new SubscriptionUpdateOptions
+        var isConsolidatedBillingEnabled = _featureService.IsEnabled(FeatureFlagKeys.EnableConsolidatedBilling);
+
+        if (isConsolidatedBillingEnabled && provider.Status == ProviderStatusType.Billable)
         {
-            CollectionMethod = "send_invoice",
-            DaysUntilDue = 30
-        };
+            var plan = StaticStore.GetPlan(organization.PlanType).PasswordManager;
+            var subscription = await _stripeAdapter.SubscriptionCreateAsync(new SubscriptionCreateOptions
+            {
+                Customer = organization.GatewayCustomerId,
+                CollectionMethod = "send_invoice",
+                DaysUntilDue = 30,
+                AutomaticTax = new SubscriptionAutomaticTaxOptions { Enabled = true },
+                Metadata = new Dictionary<string, string>
+                {
+                    { "organizationId", organization.Id.ToString() }
+                },
+                OffSession = true,
+                ProrationBehavior = StripeConstants.ProrationBehavior.CreateProrations,
+                Items = new List<SubscriptionItemOptions>
```

Comment:
⛏️  This can be a collection expression: `Items = [ new SubscriptionItemOptions { Price = plan.StripeSeatPlanId, Quantity = organization.Seats } ]`

Reply:
resolved